NatureArea.py

```diff
@@ -0,0 +1,245 @@
+from WikidataItem import WikidataItem
+import importer_utils as utils
+
+
+class NatureArea(WikidataItem):
+    """
+    Extension of WikidataItem to create nature area objects.
+
+    Handles both reserves and national parks.
+    """
+
+    def generate_ref_url(self):
+        url = "http://nvpub.vic-metria.nu/naturvardsregistret/rest/omrade/{}/G%C3%A4llande"
+        return url.format(self.raw_data["NVRID"])
+
+    def create_sources(self):
```

Comment:
The way the metadata is structured suggests these are two separate data sources. I.e. each has their own metadata entry and title, url to information page etc. So that's why I create two sources.